lib/howl/buffer.moon

```diff
@@ -10,16 +10,25 @@ ffi = require 'ffi'
 append = table.insert
 min = math.min
 
+buffer_id = 1
+
+_next_id = ->
```

Comment:
Is there a reason this is preceded by an underscore? I know that's usually done in Python modules, but here, everything's hidden except for `Buffer` itself.